MAUI/DataGrid/conditional-styling.md

```diff
@@ -382,4 +385,196 @@ public class ForeColorConverter : IValueConverter
+        <Style TargetType="syncfusion:DataGridTableSummaryCell">
+        <Setter Property="Background"
+                    Value="{Binding Converter={StaticResource converter}}" />
+        </Style>
+    </ContentPage.Resources>
+    <syncfusion:SfDataGrid x:Name="dataGrid"
+                        ShowRowHeader="True"
+                        ItemsSource="{Binding OrderInfoCollection}">
+                <syncfusion:SfDataGrid.TableSummaryRows>
+        <syncfusion:DataGridTableSummaryRow Title="Total Salary :{TotalSalary}"
+                                        Position="Bottom"
+                                        ShowSummaryInRow="True">
+            <syncfusion:DataGridTableSummaryRow.SummaryColumns >
+                <syncfusion:DataGridSummaryColumn Name="TotalSalary"
+                                                Format="{}{Sum:C0}"
+                                                MappingName="Salary"
+                                                SummaryType="DoubleAggregate">
+                </syncfusion:DataGridSummaryColumn>
+            </syncfusion:DataGridTableSummaryRow.SummaryColumns>
+        </syncfusion:DataGridTableSummaryRow>
+            <syncfusion:DataGridTableSummaryRow Title="Total Salary :{TotalSalary}"
+                                                Position="Top"
+                                                ShowSummaryInRow="False">
+                <syncfusion:DataGridTableSummaryRow.SummaryColumns>
+                    <syncfusion:DataGridSummaryColumn Name="TotalSalary"
+                                                    Format="{}{Sum:C0}"
+                                                    MappingName="OrderID"
+                                                    SummaryType="DoubleAggregate">
+                    </syncfusion:DataGridSummaryColumn>
+                </syncfusion:DataGridTableSummaryRow.SummaryColumns>
+            </syncfusion:DataGridTableSummaryRow>
+        </syncfusion:SfDataGrid.TableSummaryRows>
+    </syncfusion:SfDataGrid>
+{% endhighlight %}
+{% highlight c# %}
+public class ColorConverter : IValueConverter
+{
+    public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        var summaryValue = (value as SummaryRecordEntry).SummaryValues[0];
+        var aggregateValue = summaryValue.AggregateValues.ElementAt(0);
+        var calculatedValue = aggregateValue.Value;
+
+        //custom condition is checked.
+
+        if ((double)calculatedValue % 2 == 0)
+            return Colors.LightGreen;
+        return Colors.LightBlue;
+    }
+    public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        throw new NotImplementedException();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+Here, table summary cells are customized based on `Salary` summary value.
+
+![Conditional Style of WPF DataGrid Table Summary Cells using Converter](maui-datagrid-conditional-tablesummary.png)
```

Comment:
Conditional styling of table summary cell using converter.

Reply:
Changed